src/otx/core/exporter/base.py

```diff
@@ -211,3 +210,56 @@ def _embed_openvino_ir_metadata(ov_model: openvino.Model, metadata: dict[tuple[s
             ov_model.set_rt_info(data, list(k))
 
         return ov_model
+
+    def _extend_model_metadata(self, metadata: dict[tuple[str, str], str]) -> dict[tuple[str, str], str]:
+        """Extends metadata coming from model with preprocessing-specific parameters.
+
+        Model's original metadata has priority over exporter's extra metadata
+
+        Args:
+            metadata (dict[tuple[str, str], str]): _description_
```

Comment:
Please add the description

Reply:
done at https://github.com/openvinotoolkit/training_extensions/pull/2806/commits/241980ac6bab566316f6e9e19b541195f2771756